compiler/modules/print.llvm/llvm.bal

```diff
@@ -344,9 +344,31 @@ public class Builder {
         addInsnWithAlign(self.bb(), ["store", typeToString(ty), val.operand, ",", typeToString(ptr.ty), ptr.operand], align);
     }
 
-    // binary operation with int operands and (same) int result
-    // Corresponds to LLVMBuild{Add,Mul,Sub,SDiv,SRem}
-    public function binaryInt(BinaryIntOp op, Value lhs, Value rhs, string? name=()) returns Value {
+    // Corresponds to LLVMBuildNSW{Add,Mul,Sub}
+    public function iArithmeticNoWrap(IntArithmeticOp op, Value lhs, Value rhs, string? name=()) returns Value {
+        BasicBlock bb = self.bb();
+        string reg = bb.func.genReg();
+        IntType ty = sameIntType(lhs, rhs);
+        bb.addInsn(reg, "=", op, "nuw", "nsw", ty, lhs.operand, ",", rhs.operand);
```

Comment:
I think we should do just "nsw" here to keep the two llvm modules consistent.

Reply:
Fixed